CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
-- Add: Added URI parameter metadata=metadata-name-list for NGSIv2 retrieve requests (Issue #2544)
-- Add: Added 'metadata' as part of the payload for POST /v2/op/query (Issue #2544)
+- Add: Added URI parameter metadata=metadata-name-list for NGSIv2 retrieve requests (#2544)
```

Comment:
Given that this PR is probably going to be merged also in other branchs with the cherry-pick technique, please roll-back CNR fixes not related with the PR itself. I mean, leaving only the line about #2571